src/create-matcher.js

```diff
@@ -212,7 +212,9 @@ function fillParams (
 
 function getParams (path: string): Array<string> {
```

Comment:
Probably should rename it to `getRequiredParams` to avoid ambiguity,
or maybe remove this function and use inline expression at https://github.com/znck/vue-router/blob/c3ec53191647d5a4dfb7481ea3c4d088f2c6c6d7/src/create-matcher.js#L38